<ui version="4.0" >
 <class>SearchDialogClass</class>
 <widget class="QDialog" name="SearchDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SearchDialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" native="1" name="wdtToolBar" />
   </item>
   <item>
    <widget class="QStackedWidget" name="stwWidgets" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="pgeFields" >
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="1" column="0" colspan="2" >
        <widget class="QLabel" name="lblInstructions" >
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblFirst" >
         <property name="text" >
          <string>First:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lneFirst" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lblLast" >
         <property name="text" >
          <string>Last:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lneLast" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="lblNick" >
         <property name="text" >
          <string>Nick:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLineEdit" name="lneNick" />
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="lblEmail" >
         <property name="text" >
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="lneEmail" />
       </item>
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pgeResult" >
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tbwResult" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text" >
           <string>JID</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>First</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Nick</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>EMail</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pgeForm" />
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QWidget" native="1" name="wdtPages" />
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dbbButtons" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
